setup.py

```diff
@@ -32,7 +32,7 @@ def read_version(package):
 swagger_ui_require = 'swagger-ui-bundle>=0.0.2'
 flask_require = 'flask>=1.0.4'
 aiohttp_require = [
-    'aiohttp>=2.3.10',
+    'aiohttp>=2.3.10,<4',
```

Comment:
added < v4 because it was pulling a version 4 (tho it's not stable yet), and failing badly.